packages/react-examples/src/react-charting/HorizontalBarChart/HorizontalBarChart.ChartMode.Example.tsx

```diff
@@ -0,0 +1,116 @@
+import * as React from 'react';
+import { HorizontalBarChart, IChartProps } from '@fluentui/react-charting';
```

Comment:
Instead of adding new example, add a  toggle in basic and variant example.

Reply:
done